swupd-overdue.c

```diff
@@ -46,8 +75,8 @@ int main(int argc, char *argv[])
 	char buf[32];
 
 	if (argc == 2) {
-        exp = strtol(argv[1], NULL, 10);
-        if ( (exp == LONG_MAX || exp == LONG_MIN) && errno != 0 )
+		exp = strtol(argv[1], NULL, 10);
+		if ( (exp == LONG_MAX || exp == LONG_MIN) && errno != 0 )
```

Comment:
you should fix your first commit instead of fixing tabs/spaces in a subsequent (and unrelated) commit.